Modules/User/Routes/web.php

```diff
@@ -22,4 +22,6 @@
      * User Profile
     */
     Route::get('/profile', 'ProfileController@index')->name('user.profile');
+    Route::get('/user-settings', 'UserSettingsController@index')->name('user.settings');
+    Route::post('/user-settings', 'UserSettingsController@update')->name('user.settings-hr');
```

Comment:
This is incorrect name. Please use what the route does:
```suggestion
    Route::post('/user-settings', 'UserSettingsController@update')->name('user.settings.update');
```